lingetic-spring-backend/Dockerfile

```
@@ -1,10 +1,30 @@
+FROM amazoncorretto:23-jdk AS builder
+
+WORKDIR /app
+
+# Install findutils to provide xargs which is required by gradle but not provided by the base image
+RUN yum update -y && yum install -y findutils
```

Hadolint (2.12.0)

[warning] `yum clean all` missing after yum command.

(DL3032)